include/multipass/id_relationship_list.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2021 Canonical, Ltd.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef MULTIPASS_ID_MAP_H
+#define MULTIPASS_ID_MAP_H
+
+#include <utility>
+#include <vector>
+
+namespace multipass
+{
+typedef typename std::vector<std::pair<int, int>> id_relationship_list;
```

Comment:
Eh, that name I suggested is not good.  Since we are keeping the word "map" in the variable names, let's change it to `id_map_list`.  Don't forget to change the `#define` name to match.

Reply:
Actually, after some discussion with @Saviq, let's call this `id_mappings` to help differentiate what this actually is.  And then for the `gid_map` name, it would `gid_mappings` and similar for others.

Reply:
Hey @luis4a0,

Sorry, I guess you missed my last comment to call it `id_mappings` and such :frowning_face:

Reply:
Yes, indeed @townsend2010. GH does not update automatically in some cases and I missed that last message. Let me take care of that. Thanks!

Reply:
Done!